### DISCUSSION TITLE
Modify commit messages issued by Tina Cloud

### DISCUSSION THREAD

**Discussion by samc** [tinacms/tinacms#2582] [Ideas]
The current commit message per content mutation via Tina Cloud defaults to `Update from GraphQL client`. Would love a way to modify that commit message from the  [`Tina Cloud App`](https://github.com/apps/tina-cloud-app) bot to adhere to repo/org wide commit message standards:
- [Angular Commit Message Conventions](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-format)
- [gitmoji](https://github.com/carloscuesta/gitmoji)

A simple config option would work, but to dig in even further, would love to see a dynamic commit message / app comment that outlines / links out to affected live pages. We use  [`semantic-release`](/semantic-release/semantic-release) for release / changelog automation, and would love to field support for Tina Cloud content mutations via their [`@semantic-release/changelog`](https://github.com/semantic-release/changelog) plugin.

**Comment by ron-myers** [comment]
Completely agree @samc - you just beat me to creating the issue.  :)

From my perspective being able to:
1. override the commit message in the app code
2. dynamic commit message feature where what was changed could be seeded
3. prompt the user to confirm/override/enter the commit

IMO should also include which user made the modification(s) in that commit message.

**Reply by perkinsjr** [reply]
Thanks for additional input here. In regards to the last part about who made the commit, we had this functionality but had to temporary remove it due to some issues with Vercel.

**Comment by darenmalfait** [comment]
Yes, this would be great!

**Comment by Synomega** [comment]
+1
